devops/iam-did-auth-proxy-dev/values.yaml

```diff
@@ -151,7 +151,7 @@ did-auth-proxy-helm:
         }
 
         location /gp4btc/healthcheck {
-            proxy_pass http://gp4btc-backend-microservice.gp4btc.svc.cluster.local;
+            proxy_pass http://gp4btc-backend-microservice.gp4btc.svc.cluster.local/halthcheck;
```

Comment:
"halthcheck" doesn't seem right, or at least there is a typo

Reply:
oh, thanks a lot for noticing this!